src/gates.jl

```diff
@@ -861,6 +861,44 @@ function CRXGate(θ::Number)
     return round_complex_values(CRX)
 end
 
+@doc raw"""
+    CRXRevGate(θ::Number)
+
+Two-qubit controlled reverse [RXGate](@ref). 
+
+**Circuit Representation**
+```
+     ┌───────┐
+q_1: ┤ RX(ϴ) ├
+     └───┬───┘
+q_0: ────■────
+```
+
+**Matrix Representation**
+
+```math
+\newcommand{\th}{\frac{\theta}{2}}
+
+CRXRev(\theta)\ q_1, q_0 =
+|0\rangle\langle0| \otimes I + |1\rangle\langle1| \otimes RX(\theta) =
+    \begin{pmatrix}
+        1 & 0 & 0 & 0 \\
+        0 & \cos{\th} & 0 & -i\sin{\th} \\
+        0 & 0 & 1 & 0\\
+        0 & -i\sin{\th} & 0 & \cos{\th}
+    \end{pmatrix}
+```
+"""
+function CRXRevGate(θ::Number)
+
+    CRX = Array{Complex{Float64},2}([ 1 0 0 0            
```

Comment:
Can you name this CRXRev and on line 899. Just to be consistent, though doesn't affect the overall code.

Reply:
Fixed